lib/web/sessions.go

```diff
@@ -1042,9 +1042,6 @@ func (s *sessionCache) invalidateSession(ctx context.Context, sctx *SessionConte
 		return trace.Wrap(sessionDeletionErrs)
```

Comment:
nit: you can now just remove this `if` and return `trace.Wrap(sessionDeletionErrs)`